NGitLab/Impl/FilesClient.cs

```diff
@@ -57,6 +58,23 @@ public Task<FileData> GetAsync(string filePath, string @ref, CancellationToken c
         return _api.Get().ToAsync<FileData>(_repoPath + $"/files/{EncodeFilePath(filePath)}?ref={Uri.EscapeDataString(@ref)}", cancellationToken);
     }
 
+    public void GetRaw(string filePath, Action<Stream> parser, GetRawFileRequest request = null)
+    {
+        var url = _repoPath + $"/files/{EncodeFilePath(filePath)}/raw/";
+
+        if (!String.IsNullOrWhiteSpace(request?.Ref))
+        {
+            url = Utils.AddParameter(url, "ref", Uri.EscapeDataString(request?.Ref));
```

Comment:
We've already made sure that `request` isn't null:

```suggestion
            url = Utils.AddParameter(url, "ref", Uri.EscapeDataString(request.Ref));
```